### DISCUSSION TITLE
Sort by Milestone - Show Completion Progress option in group headings

### DISCUSSION THREAD

**Discussion by robotgryphon** [community/community#6457] [Projects and Issues]
The sort by Milestone feature is already pretty nice, but it'd be even nicer to have a visual indicator of the milestone task progress in the group headers, like how one can see it on individual issue boards.

If several milestones are enabled on a large project, this could really help people see what's close to completion, and where extra work needs to be focused.

Could we enable a tracker bar on the right side of the group headings, please?

**Comment by eddedd88** [comment]
I think you meant "group by" milestone? having the **due date** of the milestone would also be great.

I organize the work using _milestones_ and we compare _progress_ with the _due date_  to assess whether we need to change course. We want to have a view that displays all the milestones, their progress, and due date; so we can do this assessment without having to go into each milestone individually.

An extra nice functionality would be to configure a _velocity_ and _estimating_ values for a milestone so that Github would then do this assessment for us and just show in the same line whether the **Milestone** is at risk or not.
